src/Components/ArtworkGrid/__tests__/ArtworkGrid.test.tsx

```diff
@@ -0,0 +1,115 @@
+import { mount } from "enzyme"
+import { cloneDeep } from "lodash"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { renderRelayTree } from "../../../DevTools"
+import { ArtworkGridItem } from "../../Artwork/GridItem"
+import { ArtworkGridFixture } from "../__stories__/ArtworkGridFixture"
+import ArtworkGrid, {
+  ArtworkGridContainer,
+  ArtworkGridContainerState,
+} from "../ArtworkGrid"
+import { ArtworkGridEmptyState } from "../ArtworkGridEmptyState"
+
+jest.unmock("react-relay")
+global.clearInterval = jest.fn()
+
+const TestContainer = createFragmentContainer(
+  (props: any) => {
+    return <ArtworkGrid artworks={props.artist.artworks_connection} />
+  },
+  graphql`
+    fragment ArtworkGrid_artist on Artist {
+      artworks_connection(first: 4) {
+        ...ArtworkGrid_artworks
+      }
+    }
+  `
+)
+
+describe("ArtworkGrid", () => {
+  const getRelayWrapper = async (artworks = ArtworkGridFixture) => {
+    return await renderRelayTree({
+      Component: TestContainer,
+      query: graphql`
+        query ArtworkGrid_Test_Query {
+          artist(id: "pablo-picasso") {
+            ...ArtworkGrid_artist
+          }
+        }
+      `,
+      mockResolvers: {
+        Artist: () => ({ artworks_connection: artworks }),
+        ArtworkConnection: () => artworks,
```

Comment:
I was spending a long time today trying to stub out resolvers properly for a similar test elsewhere, and this made me realize I was thinking about it all wrong. Without a `ShowConnection` (in my case), I was getting the placeholder 'Hello World' and failing specs. Thank you so much for this!

Reply:
Props to @l2succes who paired with me to break out this pattern

Reply:
👏 @l2succes and @eessex . Nice work.

Reply:
this just nabbed me today too 😄